core/sql/regress/tools/runallsb

```diff
@@ -42,6 +42,7 @@ export SQLMX_REGRESS=1
 export SEABASE_REGRESS=2
 
 TEST_SUBDIRS="core compGeneral executor seabase fullstack2 charsets qat catman1 udr hive" 
+TEST_SUBDIRS="catman1 udr hive" 
```

Comment:
Sorry. I'll  restore the original version. Thanks for catching it.

On Mon, Sep 28, 2015 at 11:03 AM, DaveBirdsall <notifications@github.com>
wrote:

> In core/sql/regress/tools/runallsb
> <https://github.com/apache/incubator-trafodion/pull/94#discussion_r40569992>
> :
>
> > @@ -42,6 +42,7 @@ export SQLMX_REGRESS=1
> >  export SEABASE_REGRESS=2
> >
> >  TEST_SUBDIRS="core compGeneral executor seabase fullstack2 charsets qat catman1 udr hive"
> > +TEST_SUBDIRS="catman1 udr hive"
>
> Not sure why this change is here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/94/files#r40569992>.
>



-- 
Regards, --Qifan